zerver/lib/markdown/fenced_code.py

```diff
@@ -237,19 +250,37 @@ def done(self) -> None:
         self.processor.pop()
 
 
+def parse_hl_lines(hl_lines_str: Optional[str] = None) -> List[int]:
+    """Support our syntax for emphasizing certain lines of code.
+    hl_lines should be like '1 2' to emphasize lines 1 and 2 of a code block.
+    Returns a list of ints, the line numbers to emphasize.
+    """
+    if not hl_lines_str:
+        return []
+
+    try:
+        return list(map(int, hl_lines_str.split()))
+    except AssertionError:  # nocoverage
```

Comment:
@timabbott @andersk Should I write it as -
```python
except ValueError as e: # nocoverage
  raise AssertionError(e)
```
or as -
```python
except ValueError:
  return []
```
Latter is the upstream version.

Reply:
I think Anders' proposal was that we call the upstream function directly rather than copying it, but if that's not possible, I'd prefer to copy it exactly with a comment.